extensions/ql-vscode/src/common/commands.ts

```diff
@@ -96,6 +96,7 @@ export type LocalQueryCommands = {
   "codeQL.quickEvalContextEditor": (uri: Uri) => Promise<void>;
   "codeQL.codeLensQuickEval": (uri: Uri, range: Range) => Promise<void>;
   "codeQL.quickQuery": () => Promise<void>;
+  "codeQL.createSkeletonQuery": () => Promise<void>;
```

Comment:
Would it make sense for this to be a separate "module"? This doesn't seem like it's really related to local querying.

Reply:
I should've mentioned this in the commit message. My bad. I've discussed this with our PM and we want to make this the default way you create local queries, which is why we kept the name "CodeQL: Create Query". 

I'm happy to move it somewhere else, or alternatively we could just call it `codeQL.createQuery` or `codeQL.createLocalQuery`.